spec.bs

```diff
@@ -165,3 +165,9 @@ The {{HTMLFencedFrameElement/src}} IDL attribute must [=reflect=] the respective
 <h3 id=dimension-attributes>Dimension attributes</h3>
 
 This section details monkeypatches to [[!HTML]]'s <a href="https://html.spec.whatwg.org/multipage/embedded-content-other.html#dimension-attributes">Dimension attributes</a> section. This section will be updated to include <{fencedframe}> in the list of elements that the <code>[=width=]</code> and <code>[=height=]</code> dimension attributes apply to.
+
+<h3 id=coep>New fencedframe [=request=] [=request/destination=]</h3>
+
+Update the list of <a href="https://fetch.spec.whatwg.org/#concept-request-destination">associated destinations</a> to include a new entry, <code>fencedframe</code>. It will have the initiator <code>""</code>, the CSP directive <code>child-src</code>, and the features <code>HTML's &lt;fencedframe&gt;</code>.
```

Comment:
Looking at the convention, it looks like the quotes are usually outside of code blocks. So I think we can either just omit the code tag as you said and keep the "", or we can say "It will have no initiator". Whichever you think would be better.

Reply:
Having empty quotes sounds good